### PR TITLE
refresh flag review modal (bug 1114845)

### PR DESCRIPTION
**Pull request by ngokevin**
todo: tests

buttons to fixed later (affected globally)

screen shot 2014-12-23 at 1 53 45 am
screen shot 2014-12-23 at 1 53 50 am

depends on https://github.com/mozilla/marketplace-elements/pull/4

### CODE REVIEW

File: src/media/css/ratings.styl [hunk 1]
```diff
@@ -268,25 +268,19 @@ select[name=rating] {
...
+    li {
+        margin: 10px 0;
+
+        &:first-child {
+            margin-top: 0;
+        }
     }
-    &.show {
-        padding: 0;
+    form > div:last-child > button {
```
Comment (spasovski) [inline src/media/css/ratings.styl]: Seems a bit specific but looking at its parent it might be fine.

File: src/media/js/ratings.js [hunk 2]
```diff
@@ -23,35 +23,29 @@ define('ratings',
...
-            $('.cloak').trigger('dismiss');
-            // L10n: The report is an abuse report for reviews
-            $actionEl.text(gettext('Sending report...'));
-            require('requests').post(
-                require('settings').api_url + urls.api.sign($reviewEl.data('report-uri')),
-                {flag: $(e.target).attr('href').replace('#', '')}
-            ).done(function() {
-                notify({message: gettext('Review flagged')});
+            var $actionEl = $review.find('.actions .flag');
+            $modal[0].dismissModal();
```
Comment (spasovski) [inline src/media/js/ratings.js]: This will cause a JS error if `!$modal` we might want to `return` earlier or wrap in `if ($modal)...`

**Comment by spasovski** [comment]
r+wc though I see you have the second part up also

File: src/media/js/ratings.js (hunk 2 above)
Comment (ngokevin) [inline src/media/js/ratings.js]: when would there be ```!$modal```?

**Pull request merged**